rules/integrations/azure/privilege_escalation_azure_kubernetes_rolebinding_modified_or_deleted.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/10/18"
+integration = "azure"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies the creation or patching of potential malicious rolebinding. You can assign these roles to Kubernetes subjects 
+(users, groups, or service accounts) with role bindings and cluster role bindings.
+"""
+from = "now-20m"
+index = ["filebeat-*", "logs-azure*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Azure Kubernetes Rolebindings Modified or Deleted"
+note = """## Config
+
+The Azure Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    "https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations#microsoftkubernetes",
+    "https://www.microsoft.com/security/blog/2021/03/23/secure-containerized-environments-with-updated-threat-matrix-for-kubernetes/",
+    "https://www.microsoft.com/security/blog/2020/04/02/attack-matrix-kubernetes/",
+    "https://medium.com/mitre-engenuity/att-ck-for-containers-now-available-4c2359654bf1",
+    "https://attack.mitre.org/matrices/enterprise/cloud/",
```

```suggestion
    "https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations#microsoftkubernetes",
    "https://www.microsoft.com/security/blog/2020/04/02/attack-matrix-kubernetes/",
```
Relevant in general container security, but not specific to the rule.